yarn/src/main/scala/org/apache/spark/deploy/yarn/Client.scala

```diff
@@ -66,6 +66,8 @@ private[spark] class Client(
   private val executorMemoryOverhead = args.executorMemoryOverhead // MB
   private val distCacheMgr = new ClientDistributedCacheManager()
   private val isClusterMode = args.isClusterMode
+  private val fireAndForget = isClusterMode &&
+    sparkConf.getBoolean("spark.yarn.cluster.quiet", false)
```

Comment:
right now we don't have any configs that are .cluster. or .client.   In other places we do reference .am., .driver., etc.   That matches other spark configs that are .driver. and .executor.  So I think .client. would fit the current naming conventions where it means the client and not client mode.  That doesn't mean it couldn't be confusing to some users though.  We could try something like spark.yarn.submit.[waitForCompletion|waitAppCompletion] .  opinions on that?